tests/test_rest_api.py

```diff
@@ -89,3 +92,31 @@ def test_rest_api_will_reroute_logs_from_rpi(app_sio_rpi_client: Tuple[Flask, So
 def test_rest_api_get_username_should_return_username(client):
     response = client.get('/get_username', headers=make_auth_header('user1', 'user1'))
     print(response.json)
+
+def test_returning_all_modules_assigned_to_user(client):
+    response = client.get('/api/get_modules', headers=make_auth_header('user1', 'user1'))
+    assert response.status_code == 200
+    assert 'modules' in response.json
+
+    assert response.json['modules'] == ['unique_id_1','unique_id_3']
+
+
+#  tu jest cos nie tak z testem, problemem jest przeslanie zarowno headera z authentykacja jak i jsona w poscie
+def test_adding_module(app):
+    client = app.test_client()
+    data = { "unique_id": "unique_id_2", }
+    response = client.post('/api/add_module', headers={'Authorization': 'Basic ' + base64.b64encode(('user1:user1'.encode('utf8'))).decode('utf8'),"Content-Type": "application/json"}
```

Comment:
add `Content-Type` to `make_auth_header` function as optional parameter, like this:
```python
def make_auth_header(username: str, password: str, content_type: str = 'application/json'):
    ...
```